src/ProjectTemplates/BlazorTemplates.Tests/BlazorWasmTemplateTest.cs

```diff
@@ -250,10 +250,21 @@ private void ValidateAppWorksOffline(Project project, string listeningUri, bool
             TestBasicNavigation(project.ProjectName, skipFetchData: skipFetchData);
         }
 
-        [Theory]
-        [InlineData(true)]
-        [InlineData(false)]
-        public async Task BlazorWasmHostedTemplate_IndividualAuth_Works(bool useLocalDb)
+        [ConditionalFact]
+        // LocalDB doesn't work on non Windows platforms
```

Comment:
cc @javiercn

Reply:
Yeah, if we have to break them into two separate tests, so be it